maps/rent-burden/scripts/_application_wrapper.js

```diff
@@ -0,0 +1,141 @@
+(function(){
+
+    window.App = {
+        Models: {},
+        Collections: {},
+        Views: {},
+        Router: {}
+    };
+
+    // load external templates via ajax
+    window.template = function(url){
+        var data = "<h1> failed to load url : " + url + "</h1>";
+        $.ajax({
+            async: false,
+            dataType: "text",
+            url: url,
+            success: function(response) {
+                data = response;
+            }
+        });
+        return data;
+    };
+
+    App.Views.ApplicationWrapper = Backbone.View.extend({
+
+        el: "body",
+
+        initialize: function(){
+            // sets template path
+            if (window.location.href.indexOf("https://projects.scpr.org/") > -1) {
+                window.wrapperTemplatePath = "https://projects.scpr.org/static-files/v3-dependencies/templates/"
+            }
```

Comment:
This will cause an error when users visit the regular HTTP version of the site. We'll need another clause for that:
```
            if (window.location.href.indexOf("http://projects.scpr.org/") > -1){
                window.wrapperTemplatePath = "http://projects.scpr.org/static-files/v3-dependencies/templates/"
            }
```